test/unit/proj_context_test.cpp

```diff
@@ -179,4 +179,24 @@ TEST(proj_context, read_grid_from_user_writable_directory) {
     MyUnlink(filename);
 }
 
+// ---------------------------------------------------------------------------
+
+TEST(proj_context, proj_context_set_ca_bundle_path) {
+
+    std::string dirname;
+    auto filename = createTempDict(dirname, "temp_proj_dic2");
```

Comment:
creating the directory and deleting afterwards is probably unneeded. You could pass a completely dummy string

Reply:
Good suggestion. Thanks :+1: